CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.5.1] - 2018-10-15
+### Changed
+-- Fig bug affecting the backward expansion of possible matches starting with stop words
+
+[0.5.1]: https://github.com/snipsco/gazetteer-entity-parser/compare/0.5.0...0.5.1
+
 ## [0.5.0] - 2018-10-01
 ### Changed
-- Clearer `ParserBuilder`'s API 
+-- Clearer `ParserBuilder`'s API
```

Comment:
I think your formatter introduced some double hyphens `--`

Reply:
BTW, I think it's clearer to stack the github references at the bottom of the file